gadgets/snapshot_socket/gadget.yaml

```diff
@@ -3,36 +3,32 @@ description: Show TCP and UDP sockets
 homepageURL: https://inspektor-gadget.io/
 documentationURL: https://inspektor-gadget.io/docs
 sourceURL: https://github.com/inspektor-gadget/inspektor-gadget/
+datasources:
+  sockets:
+    fields:
+      src:
+        annotations:
+          description: Source address
+          template: l4endpoint
+      dst:
+        annotations:
+          description: Destination address
+          template: l4endpoint
+      state:
+        annotations:
+          columns.width: 10
+      ino:
+        annotations:
+          description: Inode number
+          columns.width: 10
+          columns.hidden: true
+      netns:
+        annotations:
+          description: Network namespace inode id
+          template: ns
 snapshotters:
   sockets:
     structName: socket_entry
 structs:
   socket_entry:
     fields:
```

Comment:
Do you think we'll use this `structs` field in the future? If you don't, why we're not removing it?

Reply:
I want to get this PR as soon as possible in, further cleanups will be done later on, like #3164.